docker-compose.yml

```
@@ -125,38 +143,39 @@ services:
-      - "3333:3333" # Remove in prod
-      - "80:80"
-    depends_on:
-      - mailserver
     restart: unless-stopped
     networks:
       - backend
     profiles:
       - prod
       - dev
+    post_start:
+    - command: sh -c "mkdir -p /var/lib/apt/lists/partial && which sqlite3 || (apt-get update && apt-get install -y sqlite3); if [ ! -f /opt/gophish/.initialized ]; then sqlite3 /opt/gophish/gophish.db \"INSERT INTO smtp (id, user_id, interface_type, name, host, username, password, from_address, modified_date, ignore_cert_errors) VALUES (1, 1, 'SMTP', 'smtp', '${SMTP_HOST}:${SMTP_PORT}', '${SMTP_MAIL_USERNAME}', '${SMTP_MAIL_PASSWORD}', '${SMTP_MAIL_USERNAME}', datetime('now'), '${IGNORE_TLS_ERRORS}');\" && touch /opt/gophish/.initialized; fi"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)